compass/ocean/tests/spherical_harmonic_transform/qu_convergence/mesh.py

```diff
@@ -0,0 +1,140 @@
+import numpy as np
+
+from mpas_tools.ocean import build_spherical_mesh
+from mpas_tools.logging import check_call
+
+from compass.step import Step
+from compass.model import make_graph_file
+
+import os
+
+
+class Mesh(Step):
+    """
+    A step for creating uniform global meshes
+
+    Attributes
+    ----------
+    resolution : int
+        The resolution of the (uniform) mesh in km
+
+    serial_nLat : int
+        The number of latitudes in the Gaussian grid
+    """
+    def __init__(self, test_case, resolution, serial_nLat):
+        """
+        Create a new step
+
+        Parameters
+        ----------
+        test_case : compass.ocean.tests.spherical_harmonic_transform.qu_convergence
```

Comment:
here it seems like it doesn't make sense to break this line up

Reply:
Yep, that's fine to leave as it is.  But it's not correct.  It needs `QuConvergence` at the end.

Reply:
thanks for pointing that out.

Reply:
```suggestion
        test_case : compass.ocean.tests.spherical_harmonic_transform.qu_convergence.QUConvergence
```